src/content/structured/components/accordion/guidance.mdx

```diff
@@ -0,0 +1,217 @@
+        This is an example of the main body text
+      </IcTypography>
+    </IcAccordion>
+    <IcAccordion heading="Accordion 2">
+      <SlottedSVG
+        slot="icon"
+        width="20"
+        height="1em"
+        viewBox="0 0 512 512"
+        fill="currentColor"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path d="M362.7 19.3L314.3 67.7 444.3 197.7l48.4-48.4c25-25 25-65.5 0-90.5L453.3 19.3c-25-25-65.5-25-90.5 0zm-71 71L58.6 323.5c-10.4 10.4-18 23.3-22.2 37.4L1 481.2C-1.5 489.7 .8 498.8 7 505s15.3 8.5 23.7 6.1l120.3-35.4c14.1-4.2 27-11.8 37.4-22.2L421.7 220.3 291.7 90.3z" />
+      </SlottedSVG>
+      <IcAccordion heading="Nested Accordion 1">
+        <IcTypography variant="body">
+          This is an example of the nested accordion body text
+        </IcTypography>
+      </IcAccordion>
+      <IcAccordion heading="Nested Accordion 2">
+        <IcTypography variant="body">
+          This is an example of the nested accordion body text
+        </IcTypography>
+      </IcAccordion>
+    </IcAccordion>
+  </IcAccordionGroup>
+</ComponentPreview>
+
+## When to use
+
+Use accordions to organise information, shorten the page length and reduce the scroll time when the page is initially loaded.
+
+Use accordion groups to categorise accordions into sections with a title.
+
+For example, use an accordion group for a frequently asked questions (FAQs) page.
+
+<DoDontCaution
+  imageSrc={accordionFig1}
+  imageAlt="Two collapsed accordion groups. One titled ‘Bari-star member benefits’ and one titled ‘FAQs’, both containing three accordions."
+  state="good"
+  caption="Add accordion titles to help organise content."
+/>
+
+Use the accordion title section to title the group of accordions and contain the ‘see all’ or ‘hide all’ button. Don’t display a title to achieve a title section with only the ‘see all’ or ‘hide all’ button.
+
+Use ‘see all’ and ‘hide all’ functionality to show or hide all information from all accordions at once.
+
+<DoDontCaution
+  imageSrc={accordionFig2a}
+  imageAlt="A collapsed accordion group titled ‘Bari-star member benefits’ with a ‘see all’ button."
+  state="good"
+  caption="Use the ‘see all’ button to open all accordions within an accordion group at once. "
+/>
+
+<DoDontCaution
+  imageSrc={accordionFig2b}
+  imageAlt="An expanded accordion group titled ‘Bari-star member benefits’ with a ‘hide all’ button."
+  state="good"
+  caption="Use the ‘hide all’ button to close all accordions within an accordion group at once."
+/>
+
+Add custom content to an accordion using the slotted section to create different uses for your accordions.
+
+<DoDontCaution
+  imageSrc={accordionFig3}
+  imageAlt="An accordion titled ‘Order details’ including a three-row data entity for a coffee order in the content area."
+  state="good"
+  caption="Use the slotted content of an accordion to house other content."
+/>
+
+Set an accordion to open one at once when the information in the accordions is unrelated.
```

Comment:
I think this makes more sense as @MI6-94 says:
```suggestion
Set an accordion to open one at a time when the information in the accordions is unrelated.
```
Would be happy to approve after this is updated :)